templates/shared/_latest-news.html

```diff
@@ -0,0 +1,16 @@
+<script src="{{ versioned_static('dist/latest-news.js')}}"></script>
```

Comment:
This script is included before the HTML elements it requires.

It works fine only because it renders anything after http response, so execution is delayed.
But it's a good practice to have scripts run after the HTML they need (to make sure it's available at the time script runs). Ideally it should be at the bottom of the page in a common section for all the scripts.

Reply:
Thanks @baraz, I agree the script tag should ideally be after the markup it uses, just to be sure (though in this case I think it will definitely work every time - the HTTP request will always take many orders of magnitude longer than processing the markup).

Note that there is a significant difference between placing the script tag directly after the markup and placing it at the end of the page - a choice we should make deliberately. If it's directly after the markup, the HTTP request will be sent right after the markup is processed. If it's at the end, this HTTP request could be sent after the browser encounters many more images/script tags etc. Since this "latest news" element appears as core content of the page, an argument could be made that we really do want to optimise for grabbing that content as soon as possible.

I don't have a very strong opinion either way, but we need to be aware of this caveat when moving scripts to the end of the page.

Reply:
I totally get the performance gain of having this close to markup to make it load news as soon as possible and I'm fine with keeping it up for performance gain.
Still, I think it would be good at least to move it under the HTML it requires, so in case the script is ever modified to for example hide/show some elements before loading, we can be sure it will work.

Reply:
I have moved it below now. I agree with @bartaz  that the likelyhood of fogetting that this is loaded at the top is bigger than the few MS we gain in performance in this case.